client/my-sites/customer-home/main.jsx

```diff
@@ -79,7 +79,7 @@ const Home = ( {
 			<FormattedHeader
 				brandFont
 				headerText={ translate( 'My Home' ) }
-				subHeaderText={ translate( 'Your home base for posting, editing, and growing your site.' ) }
+				subHeaderText={ translate( 'Your hub for posting, editing, and growing your site.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.